libnavui-shield/src/test/java/com/mapbox/navigation/ui/shield/RoadShieldContentManagerImplTest.kt

```diff
@@ -400,7 +408,7 @@ class RoadShieldContentManagerImplTest {
             expectedShield,
             RouteShieldOrigin(
                 isFallback = false,
-                originalUrl = toDownload.url,
+                originalUrl = toDownload.originalUrl,
```

Comment:
The goal of the `originalUrl` is to provide an executable URL that developers can call and see what's wrong. Can we keep this as `originalUrl = toDownload.url`?

Reply:
I made this change because of the comparison required here

```
private fun getShieldToRender(
        node: RoadShieldComponentNode,
        roadShields: Set<RoadShield>?
    ): RoadShield? {
        return roadShields?.find {
            ifNonNull(node.mapboxShield) { mbxShield ->
                val shieldName = mbxShield.name()
                val displayRefLength = getRefLen(mbxShield.displayRef())
                val shieldRequested = shieldName.plus("-$displayRefLength")
                it.shieldUrl.contains(shieldRequested)
            } ?: false
        } ?: roadShields?.find {
            it.shieldUrl == node.shieldUrl
        }
    }
```
Since the set can now contain different shields, I need to find if the shield url from RouteShieldResult matches with the shield url in the directions response. The direction response `legacyURL` does not contain `.svg`, but we add it here `override val url: String = initialUrl.plus(SVG_EXTENSION)` to be able to make network request. In order to do the comparison here I would have to remove the `.svg` extension from the RouteShieldResult url obtained. I don't like that approach, because if something gets added in the API in the future, then we will have to remove it from the view which has the potential to mismatch.

@LukasPaczos

Reply:
I see the ask was for `RouteShieldOrigin`. The changes have been already been reverted.